dbrequests/connection.py

```diff
@@ -106,13 +107,26 @@ def _send_data_pandas(self, df, table, pandas_mode='append', **params):
                   index=False, **params)
 
     @contextmanager
-    def _temporary_table(self, table):
+    def _temporary_table(self, table: str, with_cols: (str, None) = None):
         tmp_table = 'tmp_dbrequests_' + table
         self.bulk_query('''
         create temporary table `{tmp_table}` like `{table}`;'''.format(
             tmp_table=tmp_table,
             table=table
         ))
+        if with_cols is not None:
+            # we can drop columns not being useed
+            res = self.query('show columns from {table};'.format(table=table))
+            cols_to_drop = [
+                k for k in res.Field.to_list() if k not in with_cols]
+            if len(cols_to_drop) > 0:
+                drop_query = 'alter table `{tmp_table}` {cols_to_drop};'.format(
+                    tmp_table=tmp_table,
+                    cols_to_drop=', '.join(
+                        ['drop column `' + col + '`' for col in cols_to_drop])
```

Comment:
```suggestion
                        ['drop column `{}`'.format(col) for col in cols_to_drop])
```

Willms hat noch eine neuere Variante zum Strings-Basteln erwähnt, die ich aber gerade nicht finde.

Reply:
Ich bin gespannt. Da das aber überall in diesem Paket so umgesetzt ist, würde ich das nur ungern aufbrechen.